### PR TITLE
Balance View to testing the lightningd notifications 

### PR DESCRIPTION
**Pull request by vincenzopalazzo**
How discussed in #22 I split the commit and this commit represents the new Main UI to see if the UI notification work.

I will answer here to some your point in #22 

>I think the UI contains twice the same actions, accessible and displayed in different ways:
build invoice:
from menu: sats icon
from receive button: open same dialog as before

Do you think that is possible remove the item from the menu? is more intuitive the button in the main view?

>read invoice:
from send button: manually typing a invoice (I think that will never happen) or copy from clipboard
from camera floating bar: scan qr code or copy from clipboard

I agree with this point, I make this config because sometimes the people could receive the bolt11 from URL and in these cases we need somethings to display this info.
In addition, we have now a good qr scan to catch different type url, for this reason, I don't insert the qr inside the pay invoice UI. Inside the pay UI the QR should be accepted only the bolt11 (or bitcoin address)

>PS: send / receive are common words used both for onchain & offchain tx. Maybe it is only my opinion but I prefer pay (instead send) and new invoice (instead receive).

I agree, but maybe "instead send" is too long inside some display? :smile:

>I am pretty sure most of the previous UI will be overwritten as soon as an organic design will be available, but I prefer to merge the file logger contained here asap.

Oh yes, I made this UI improvement only to test the notify and maybe a UI with more buttons or with an UI like other wallets on mainet can help us to make the app accessible to other people that can help us to make more tests. 
In other words I'm not proposing the official UI :smile:

### CODE REVIEW

**Comment by lvaccaro** [comment]
Yeah I see.. Balances are a huge improvement in the main view and UI could be improved later..
A last question (sorry to be late, I have to tell before): for now lamp icon shows the current state of clightning, and also it shows the progress during pending turning on/off..
Do you think you could keep it in this UI? I am ok if you want to remove other stuff or resize the icon itself.

**Comment by vincenzopalazzo** [comment]
>Do you think you could keep it in this UI? I am ok if you want to remove other stuff or resize the icon itself.

I can try, I think that icon needs a little work to optimize the space inside the UI, but I agree with the option to maintains the lamp on Main View.

I will fix the conflicts today in this PR

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,80 +1,79 @@
...
-    <ImageView
-        android:id="@+id/arrowImageView"
-        android:layout_width="match_parent"
-        android:layout_height="50dp"
-        android:layout_gravity="bottom|center"
-        android:tint="@color/colorPrimary"
-        app:srcCompat="@drawable/ic_arrow"
-        android:visibility="gone" />
+        <include
+            android:id="@+id/content_main_status_on"
```
Comment (lvaccaro) [inline app/src/main/res/layout/activity_main.xml:50]: content_main_status_on is defined twice

**Comment by lvaccaro** [comment]
Sorry to be quite annoying, on my simulator I still have UI issue.

Screenshot_1597874453

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (vincenzopalazzo) [reply app/src/main/res/layout/activity_main.xml:50]: Sorry about this refused.

**Comment by vincenzopalazzo** [comment]
@lvaccaro, sorry about that issue with a small display. the change inside my commit yesterday what make the UI scrollable. You can scroll the UIto see the QR code. But for me is not enough, and I like how the UI looks like now.

But, yesterday after the commit I don't write any message to you because I want try to convert the lamp from png icon to an svg. I think that we can use the android system to load different icons with different dimensions to make the lamp icon on a small display smaller. What do you think about this idea?

This idea with the svg icon is easier because we can use android studio to create icon with different size by the display

if you like this idea, I want ask if you have some original source of the lamp png icon, to convert this in svg with the native source.

Or I'm losing somethings?
